src/ui/components/Header.tsx

```diff
@@ -58,7 +60,18 @@ export function Header({
       </Navbar.Brand>
       <Navbar.Content>
         <Navbar.Item>
-          <div className="flex justify-start gap-2 md:justify-end">
+          <div className="flex items-center justify-start gap-2 md:justify-end">
+            {currentInfo === 'true' && (
+              <button
+                className="flex h-fit w-fit cursor-pointer items-center justify-center rounded-full bg-white p-2"
+                onClick={() => {
+                  window.location.reload();
```

Comment:
Do this force the full page to reload? If so, could we close/remove the element in place with DOM manipulation?